docs/source/ops/system_architecture_faq.rst

```diff
@@ -0,0 +1,89 @@
+
+The short answer is that Daml Ledger data lives both on the Canton participant nodes and on the sequencer backend, meaning the blockchain or database enabled by the driver. The data is stored in the two places in different ways, but remains fully consistent thanks to Canton's deterministic execution model.
+
+All communication between Canton nodes, including the confirmation requests for transactions and the resulting confirmations and rejections, are stored on the sequencer backend. Since Daml and Canton are built around deterministic execution, you can thus consider that data on the sequencer backend, together with the Participant Encryption Keys, to be a complete copy of the Daml Ledger. 
+
+On the flip side, each Participant node stores its view of the Daml Ledger in an unencrypted, indexed, and queriable format. The set of all participant nodes jointly holds the entire ledger state and history.
+
+**Further Reading:**
+
+- `Transaction processing in Canton <../canton/architecture/overview.html#transaction-processing-in-canton>`__
+
+
+How is Canton able to recover from data loss?
+/////////////////////////////////////////////
+
+As discussed in ":ref:`where_holden_source`", the ledger data lives in two places, once encrypted in the sequencer backend, and once unencrypted spread between participant nodes. As long as you have a complete copy in either place, you can recover and continue operation.
+
+As long as the entire ledger history in the sequencer backend is available, and you hold a participant's Participant Encryption Keys, it is possible to recover the participant from the underlying sequencer backend. So if you use a blockchain as the sequencer backend, and can ensure that that blockchain stays available and uncorrupted, you can always recover from participant data loss.
+
+Should your sequencer backend go down, but all participants are still up and running, you can continue running the system by coordinating all participant nodes to “transfer” active contracts to a new domain, with a new sequencer backend.
+
+Should the sequencer backend no longer have the full ledger history, for example due to a domain switch, or because of deliberate ledger pruning, participants can still recover from a combination of the partial sequencer backend and a state snapshot. Such a snapshot can come either from a backup, or from the participants' peers.
+
+To be able to get snapshots from peers securely, nodes  regularly exchange “commitments” via the underlying blockchain. You can think of these as hashes of shared state. If Alice and Bob each run a participant, Alice's participant will regularly communicate a hash of the state it shares with Bob's participant and vice versa. This provides real-time consistency checks, but also helps recovery in the above scenario. Alice can ask Bob for a snapshot of her data shared with Bob, and check its correctness by comparing it to the commitment she made on the sequencer backend.
```

Comment:
```suggestion
To be able to get snapshots from peers securely, nodes  regularly exchange “commitments” via the underlying blockchain. You can think of these as hashes of shared state. If Alice and Bob each run a participant, Alice's participant will regularly communicate a hash of the state it shares with Bob's participant and vice versa. As the state is the same, the hash will be the same. This provides real-time consistency checks, but also helps recovery in the above scenario. Alice can ask Bob for a snapshot of her data shared with Bob, and check its correctness by comparing it to the commitment she made on the sequencer backend.
```

Reply:
You might want to add that this is currently only supported theoretically through a manual process. I.e. we have the commitments, but we don't have the automated contract synchronisation and exchange.